raiden/api/rest.py

```diff
@@ -272,6 +274,8 @@ def deposit(self, token_address, partner_address, amount):
             )
         except EthNodeCommunicationError as e:
             return make_response(str(e), httplib.REQUEST_TIMEOUT)
+        except InsufficientFunds as e:
```

Comment:
Please add a test in the API for this case with insufficient funds. Note, the API completely mocks the python api part and it just checks that the `rest.py` logic is correct.